docs/spec/spec.md

```diff
@@ -95,54 +94,53 @@ Trigger._
 
 ## kind: Channel
 
-### group: eventing.knative.dev/v1alpha1
+### group: messaging.knative.dev/v1alpha1
 
 _A Channel logically receives events on its input domain and forwards them to
-its subscribers._
+its subscribers. A custom channel implementation (other than the default _InMemoryChannel_)
+can be referenced via the _channelTemplate_. The concrete channel CRD can also be instantiated directly._
 
 ### Object Schema
 
 #### Spec
 
 | Field                    | Type                               | Description                                                                | Constraints                            |
 | ------------------------ | ---------------------------------- | -------------------------------------------------------------------------- | -------------------------------------- |
-| provisioner\*            | ObjectReference                    | The name of the provisioner to create the resources that back the Channel. | Immutable.                             |
-| arguments                | runtime.RawExtension (JSON object) | Arguments to be passed to the provisioner.                                 |                                        |
+| channelTemplate          | ChannelTemplateSpec                | Specifies which channel CRD to use.                                        | Immutable                              |
```

Comment:
It looks like we don't define `ChannelTemplateSpec` anywhere?

Reply:
is in apis/duck

Reply:
I defined the `ChannelTemplateSpec` in `Shared Object Schema`. Thank you for pointing this out :)